selfdrive/thermald/thermald.py

```diff
@@ -203,30 +202,30 @@ def thermald_thread():
     params.put_bool("BootedOnroad", True)
 
   while 1:
-    pandaState = messaging.recv_sock(pandaState_sock, wait=True)
+    sm.update(pandaState_timeout)
```

Comment:
hmm, probably doesn't change much, but do we really want to conflate here?

Reply:
I'll change it back to a regular socket, then I also don't have to do the other check